.nuspec/maui.aotprofile.txt

```diff
@@ -1671,33 +1645,30 @@ Methods:
 	Microsoft.Maui.Controls.BindablePropertyKey Microsoft.Maui.Controls.BindableProperty:CreateReadOnly (string,System.Type,System.Type,object,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.BindableProperty/ValidateValueDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangedDelegate,Microsoft.Maui.Controls.BindableProperty/BindingPropertyChangingDelegate,Microsoft.Maui.Controls.BindableProperty/CoerceValueDelegate,Microsoft.Maui.Controls.BindableProperty/CreateDefaultValueDelegate)
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.AppThemeBinding:Clone ()
-	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Binding:Clone ()
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Xaml.AppThemeBindingExtension:Microsoft.Maui.Controls.Xaml.IMarkupExtension<Microsoft.Maui.Controls.BindingBase>.ProvideValue (System.IServiceProvider)
 	Microsoft.Maui.Controls.BindingBase Microsoft.Maui.Controls.Xaml.BindingExtension:Microsoft.Maui.Controls.Xaml.IMarkupExtension<Microsoft.Maui.Controls.BindingBase>.ProvideValue (System.IServiceProvider)
-	Microsoft.Maui.Controls.BindingMode Microsoft.Maui.Controls.BindingBase:get_Mode ()
 	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.Brush:get_Default ()
 	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.Brush:op_Implicit (Microsoft.Maui.Graphics.Color)
+	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.NavigationPage:get_BarBackground ()
 	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.RadioButton:ResolveThemeColor (string)
 	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.Shell:get_FlyoutBackground ()
 	Microsoft.Maui.Controls.Brush Microsoft.Maui.Controls.VisualElement:get_Background ()
-	Microsoft.Maui.Controls.ColumnDefinition Microsoft.Maui.Controls.DefinitionCollection`1<Microsoft.Maui.Controls.ColumnDefinition>:get_Item (int)
 	Microsoft.Maui.Controls.ColumnDefinitionCollection Microsoft.Maui.Controls.Grid:get_ColumnDefinitions ()
 	Microsoft.Maui.Controls.ControlTemplate Microsoft.Maui.Controls.TemplatedPage:get_ControlTemplate ()
-	Microsoft.Maui.Controls.DataTemplate Microsoft.Maui.Controls.BaseShellItem:CreateDefaultFlyoutItemCell (string,string)
 	Microsoft.Maui.Controls.DataTemplate Microsoft.Maui.Controls.Internals.DataTemplateExtensions:SelectDataTemplate (Microsoft.Maui.Controls.DataTemplate,object,Microsoft.Maui.Controls.BindableObject)
-	Microsoft.Maui.Controls.DataTemplate Microsoft.Maui.Controls.Platform.Compatibility.ShellFlyoutRecyclerAdapter:get_DefaultItemTemplate ()
-	Microsoft.Maui.Controls.DataTemplate Microsoft.Maui.Controls.Platform.Compatibility.ShellFlyoutRecyclerAdapter:GetDataTemplate (int)
-	Microsoft.Maui.Controls.DataTemplate Microsoft.Maui.Controls.Shell:Microsoft.Maui.Controls.IShellController.GetFlyoutItemDataTemplate (Microsoft.Maui.Controls.BindableObject)
```

Comment:
I was looking at the methods that were removed here, did we maybe lose a layout pass that was being recorded before?

Reply:
Fixed
I also noticed that we lost some animation calls if you run on a simulator vs physical so I reran it on a physical device and seems mostly additive now.